3-get-session-info.md

```diff
@@ -0,0 +1,65 @@
+# Call Kobiton REST API to get session information
+This part will demonstrate how to get session information with Kobiton rest api. 
+>Kobiton have supported multiple language for [API documentation](add_api_doc_url). Go to docs for further language support
+
+## Prerequisites
+  - Kobiton account.
+
+## 1. Authentication
+To make a request:
+- Encode your credentials in base64 for HTTP Basic Authenitication. You can run this command below to get the encoded token:  
+`echo -n <your_username>:<your_api_key>`
+
+
+## 2. Get session information through Kobiton REST API
+- To send an API request:
+  ```
+  curl -X GET https://api.kobiton.com/v1/{request_path}
+  -H 'Authorization: Basic dGVzdHVzZXI6MTIzZWQtMTIzZmFjLTkxMzdkY2E='
+  -H 'Accept: application/json'
+  ```
+
+### 2.1 Get Application Info
+`GET /apps/{application_ID}`  
+You can get your application id in your desiredCaps.
```

Comment:
"id" -> "ID"
I think it looks more professional. And it's an acronym so it should be capitalized anyway.